maestrowf/specification/__init__.py

```diff
@@ -0,0 +1 @@
+__all__ = ("YAMLSpecification")
```

Comment:
You also need an import here for `from .yamlspecification import YAMLSpecification` -- you can then update the import in `maestro.py` to just `from maestrowf.specification import YAMLSpecification`.